boards/arm/stm32373c_eval/doc/stm32373c_eval.rst

```diff
@@ -148,12 +146,6 @@ Then, enter the following command:
 
    $ make BOARD=stm32373c_eval flash
 
-Run a serial host program to connect with your STM32373C-EVAL board:
```

Comment:
why remove this?

Reply:
Because it's not needed to test blinky sample, it does'nt print over serial, so, if you do this, you will see nothing.